### DISCUSSION TITLE
pillow

### DISCUSSION THREAD

**Discussion by yunguangli** [flet-dev/flet#3207] [Packages] [answered]
https://pypi.org/project/pillow/#files

**Comment by FeodorFitsner** [comment, marked as answer]
[`pillow`](https://pypi.flet.dev/pillow) and [other packages](https://pypi.flet.dev/) are available in Flet 0.25.0 pre-release and up for both iOS and Android. [Give it a try](https://flet.dev/blog/flet-new-packaging-pre-release) and let us know how that works.

**Reply by paul-upfeat** [reply]
hi, is it possible to update this for pillow>=11.0.0?

**Reply by FeodorFitsner** [reply]
On it!

**Reply by FeodorFitsner** [reply]
pillow 11.1.0 is available for both iOS and Android!

**Reply by paul-upfeat** [reply]
thank you very much for the quick response!
